### ISSUE TITLE
Week of 9/9 #hqhuddle

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#642]
@lzim @jessfroe @ritahitching morning hq! 
**Cross-ref: #613**

@staceypark 
- [x] mtl.how/demo links: mtl live & mtl video intro update links
- [x] pull requests with Rita for updating bio
- [ ] schedule manuscript planning meeting with @lzim 
- [ ] make sure all SOP edits are updated in OSF

_gifs_
- [x] give example of how to do it, and try to do a gif (with new monitors) based on Jane's comments
- [x] update guides and update the cards as an example
- [ ] then train orient

_training_
- [x]   train @jessfroe & @ritahitching on OSF file check in/check out; done with @ritahitching 

_cross-ref PR #636_
- [x] Can we please document this for the team, so that everyone is clear.
- [x] It may be something that goes in Team PSD > Resources > GitHub guide (re: Pull Requests)?

@ritahitching
- [x]  back up 8/30 adobe connect session 
- [x]  rename adobe connect recordings according to our new naming convention
- [x]  link issue #303 in the top of each _MTL Facilitate_ Training in **Lucid** 
- [x] label inside and outside of the box (number and serial number) and take a photo - 16 monitors - create a spreadsheet & send to NCPTSD Admin (Doug & Mary) as well
- [x] document ports for monitors - touch screen - requires a USB and video chord - check HDMI and SVGA 
- [x] test all monitors with commensurate stations of mtl team
- [x] use stations that are mapped consult with @staceypark 
- [x] add the #hq meeting attendance details (aka divvying & differentiating HQ roles) to the **training project** of OSF: https://osf.io/adfx3/

_training_
- [x]   @ritahitching add/update training needs in the manual

- [x] We also need to start to circulate details about the **Team PSD fall party.** 
- [x] We will likely want to send out a separate invitation, but also may want a Calendar icon or some other way to denote upcoming events in the team.

@jessfroe 
- [x] Need to remove the "Dr." from the _MTL Facilitator_ list and just alphabetize by first name (for today)
- [x] Need to populate team 2 and 4 for s06 _only_ today
- [x] **tt_teampsd:** Our celebration should be the completion of the **MTL 1.8 release milestone.**
- [ ] **FIDELITY:** finish **fidelity RedCap**, 1-pagers "**say_checklists**," **team_tracker**, **qual_workgroup**.
- [x] Qual coding for meeting tomorrow morning
- [x] Reschedule VAPOR for 9/24 separate from the Grants/Co-I meeting.
- [x] @jessfroe In **Lucid** Re-name the 3 weekly Facilitate Sessions (M/W/F) according to what will be covered that week (s07: _Modeling to Learn_ base case)

_training_
- [ ]  Add hq manual to OSF

**Comment by lzim** [comment]
Great!

Thanks @staceypark 🤘🏾

Happy Monday HQ @ritahitching and @jessfroe 😎

**Comment by lzim** [comment]
> Great!

Thanks @staceypark 🤘🏾

Happy Monday HQ @ritahitching and @jessfroe 😎
 
After Workgroup leads, I’m going to try hard to make it an all day #manuscriptMonday day for me

**Comment by jessfroe** [comment]
@ritahitching @lzim @staceypark good morning 🌞 

I'll add that my priorities today 9/9 are
- [ ] Qual fidelity rating operational definitions 
AND
- [ ] Qual coding for meeting tomorrow morning

Also, FYI, I have a doctor's appt **tomorrow** that was unexpectedly rescheduled in San Jose at 2:15p. It shouldn't take long, so I can continue working after the appointment if there are items I am able to complete remotely. 

I am also out this Friday 9/13 for the GRE. 😮

**Comment by jessfroe** [comment]
I'm also linking #613 so we can easily access the previous huddle. In case we want to reference it for any reason.

**Comment by lzim** [comment]
# HQ Decisions Re: #hqhuddle and priorities               

## 1. @staceypark "week of" will 
- Use point person headers
- (optional) of sub-headers of possibly related tasks, e.g., "training" 

## 2. Number in order of priority - (top 3 things that need to be done by lunch)
- Don't worry about anything (ordering of priorities, etc.) that doesn't have to be done by lunch.

## 3. Re-assess at the mid-day #hqhuddle
 - Edit the issue to consolidate lists.

## 4. Re-assess at the mid-day #hqhuddle
 - Edit the issue to consolidate lists.

**Comment by lzim** [comment]
Hi HQ!

I wanted to give you all a heads up pre-lunch, that I've squeezed in a meeting with Craig and Tom from 12noon-1PM.

- Please either: 1) mid-day #hqhuddle without me, or 2) skip the huddle and proceed with what we discussed this morning.

- I'll be ready for a COB #hqhuddle at 4PM.

Thanks!

Lindsey

**Comment by jessfroe** [comment]
Thanks for the heads up, @lzim 
@staceypark @ritahitching I am okay working on the items we've already discussed and huddling at 4:00p. Let me know what you think.

**Comment by ritahitching** [comment]
### ***Screen Tracking Details***

- Photos sent via email to LZ 
- cc NCPTSD Admin Doug & Mary - @lzim do you want me to email them?
- Update as monitors are tested
- Screens supplied with HDMI & VGA ports & Video & USB

| Box Number  | Serial Number |User Name|Tested|
| ------------- | ------------- | ------------- |------------- |
| 1 | K1LMTF136062 |LZ  | Yes|
| 2 | K1LMTF136061  |LZ|Yes|
| 3 | K1LMTF135441  |SP | |
| 4 | K1LMTF135404  | SP| |
| 5 | K1LMTF135442  | SH  | |
| 6 | K1LMTF135444  |RH  | |
| 7 | K1LMTF135421  |TBC| |
| 8 | K1LMTF135414  | RH  | |
| 9 |  K1LMTF1 35454 | TBC  | |
| 10| K1LMTF135457  | JF | |
| 11 | K1LMTF135434  | TBC  | |
| 12 | K1LMTF135437  | JF  | |
| 13 | K1LMTF135980  | TBC  | |
| 14 | K1LMTF135471  | SH  | |

**Comment by jessfroe** [comment]
@staceypark @lzim 
When you say rename the M/W/F meetings to correspond to the weekly material, does that mean instead of "Manuscript Monday", "Intersession Prep", and "MTL Live Co-facilitation Practice", all of the meetings would be named "s07: _Modeling to Learn_ base case"?
Or, would it be "Manuscript Monday - S07, _Modeling to Learn_ base base"?
Or, something I am not even thinking about?

**Comment by staceypark** [comment]
@jessfroe @lzim I think this would make the most sense to me:
#manuscriptmonday - s07: Modeling to Learn base case
#fidelityfriday - s07: Modeling to Learn base case

our poor wednesday still has no alliterative name though

**Comment by jessfroe** [comment]
@staceypark I do like using the hashtags! I can start incorporating those at least on Mon/Fri.

**Comment by ritahitching** [comment]
### **Meetings** Attendanc

| Monday  | Tuesday |Wednesday| Thursday| Friday|
| ------------- | ------------- | ------------- |------------- | ------------- |

**Comment by lzim** [comment]
> @jessfroe @lzim I think this would make the most sense to me to reinforce our hashtags and clarify which session we are doing:

**#manuscriptmonday - s07: Modeling to Learn base case**
**#fidelityfriday - s07: Modeling to Learn base case**
> 
> our poor wednesday still has no alliterative name though

@jessfroe I agree with @staceypark 

Let’s put the session details at the end after the session we’ll be focusing on, like in the bolded examples above. **_Please do the same for Wednesday, it’s ok that it does haven’t a catchy hashtag_**. 😢

Why? **We WILL be keeping #intersessionprep meetings in the future, but it just won’t always be on Wed.**, because the facilitators will be meeting on the “off week” whenever they normally meet with their team.

Although the training will end, we likely will be keeping #manuscriptmonday and #fidelityfriday in some form in perpetuity 📆

**Comment by lzim** [comment]
@staceypark 

- [ ] Alert GitHub to problems with PR and Issues with the Inspector Console screen shots

**Comment by ritahitching** [comment]
Resources Wish List
- To be added to Resources after initial list is complete

@lzim @staceypark @jessfroe 
**Books

**Comment by jessfroe** [comment]
@lzim @staceypark 
Sounds great, thanks for the feedback. I will start populating the meetings with the new titles tomorrow. 
👍🏼

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by staceypark** [comment]
- [ ] Get Steve Lindley a github account

**Comment by jessfroe** [comment]
@lzim @staceypark @ritahitching 
If we are huddling after lunch at 1:30, I will miss the huddle. Sorry about that. 
Please include any tasks for me on the thread and I will turn to them after my doctor's appointment. 

I am working on changing the names of the M/W/F meetings, as well as editing the S07 checklist a bit.

**Comment by ritahitching** [comment]
@lzim @staceypark @jessfroe 
Show link to training manual - so that I can update / add content, please.

**Comment by lzim** [comment]
@lzim 

- [ ] 1. Dexter's Demo Account deleted [again] set him up with course code, or talk with @staceypark about pros of adding him to one of our _MTL Facilitate_ training teams.
- SIRC2019

- [ ] 2. Once Dexter creates a free GitHub log-in; add him as a Team PSD Contributer

- [ ] 3. Have Dexter sign the NCPTSD Research Scope of Practice

- [ ] 4. Test Data UI permissions at next meeting

**Comment by lzim** [comment]
# Mid-day #hqhuddle 

@staceypark 
- [ ] Tracking **quant_workgroup** - need details and cost estimate re: **EBP reach continuity**
- Need to confirm that there isn't any wrinkles/changes that shift in any way from what proposed 

_R01/IIR Launch_ Science
- [ ] Manuscripts **R01/IIR Protocols**
- **R01** https://osf.io/8pzxu/
- **IIR** https://osf.io/p7g8f/

- [ ] **Create a Checklist:** Review what we have/don't have for the CONSORT-CRT Extension **CONSORT Wiki: https://osf.io/z7y8m/wiki/consort/**
- [ ] R01 CT.Gov and ART registration **Cross-ref** #606 and #607 
- **r21_r01_iir_procedures** https://osf.io/z7y8m/

@jessfroe 
- [ ] Translate the **s07_checklist** into a **team_tracker** **s07** checklist 
- produce a prototype if there bold in the "1-pager" it should be in the list for the _MTL Facilitate_ to tailor to the team

@ritahitching 
- [ ] Team PSD Training Guide in OSF **hqtraining** https://osf.io/adfx3/

**Comment by lzim** [comment]
@staceypark @jamesmrollins @jessfroe @ritahitching 

- I don’t want the training re: TEAM and DEMO sites to get lost...
- Based on the issues I had with Dexter today, I continue to feel we need a full DEV > TEST > PROD for DEMO...

@staceypark Although I’ve wanted to get to it.
- [ ] I still haven’t updated the MTL SEE guide link on https://mtl.how/demo which should point to https://mtl.how
- [ ] We also need the updated **video** link for s01, it’s a broken link to the old version at present.

**Comment by lzim** [comment]
> ### **Meetings** Attendance
> @lzim @staceypark @jessfroe - please edit and I'll then add to the training project of OSF: https://osf.io/adfx3/
> 
> * not sure I got all the attendees correct
> 
> 8:00-9:00	9:00-12:00
> **Monday**	workgroup leads	manuscript monday
> Attend	LZ RH SP JF	LZ RH SP
> 8:00-9:30	10:00-10:55	11:00-11:55	12:00-12:55
> **Tuesday**	qual coding workgroup	manuscript group	psychotherapy tool	psychotherapy tool
> Attend	LZ SP	LZ SP RH	LZ SP RH	LZ SP RH
> 9:30-11:30	11:00- 12:00
> **Wednesday**	intersession prep	all staff meeting
> Attend	LZ SP RH	LZ RH SP SM SH JF
> 8:00-8:55	10:00-10:55	1:00-2:00
> **Thursday**	support workgroups	qant workgroups	HQ workgroup
> Attend	LZ SP	LZ SP RH	LZ SP RH JF
> 9:30-11:30
> **Friday**	MTL Facilitate
> Attend	LZ RH SP JF

@ritahitching 

Can you format this like a typical weekly calendar:

| Monday | Tuesday | Wednesday | Thursday | Friday
| - | - |- | - | -
| meeting | meeting | meeting | meeting | meeting

**Comment by ritahitching** [comment]
@lzim re: meeting attendance - I'll reorder

**Comment by lzim** [comment]
**HQ** @staceypark @ritahitching @jessfroe
- **OLD Business please add for tomorrow’s Support Workgroup leads meeting**

_Re: SOP, Workgroup Leads, Cost (time) estimates & Due Dates_

- Stacey, I believe the conversation we had with Savet yesterday was productive, and illuminated a few aspects of the SOP that really need to be nailed down.
- I have recommendations that I believe will reduce issues that are stressful for all parties. 

Specifically, I propose:

1. **_trackers_ require current/accurate cost estimates in order to evaluate TeamPSD dependencies and priorities**
- All **COST (Time) estimates** need to be provided on issue cards as part of the triage process. 
- No issue should move forward thru the kanban process without cost (time) estimate details necessary for managing workflows smoothly.
- Any points of _clarification that influence cost (time) details_ must be discussed at **Workgroup Leads Meetings** on Mon and/or Thursday before they are changed/updated.
 - Any points of _revision/edits that influence prior cost (time) estimates_ must be discussed at **Workgroup Leads Meetings** on Mon and/or Thursday before they are changed updates

_PURPOSE of SOP Recommendation 1:_ 
a. HQ cannot provide accurate guidance on priorities without cost (time) information.
b. All Team PSD Workgroup leads benefit from accurate situational awareness about their own Workgroup, and other workgroups’, cost (time) estimation details.
c. This is key for not only identifying dependencies, which is increasingly on everyone’s radar, but also _for identifying worksteams that can occur in parallel_ so that there aren’t unnecessary lags/delays due to unnecessary serial work.
 
2. **_Due Dates_ on issues should not be edited on the kanban/tracker outside of a **Workgroup Leads, Support Workgroup Leads, or Workgroup meeting** (with only rare exceptions discussed/documented on the GH issue)
- It appears that the DUE DATES are currently meaningless.
- Example 1: Andrew did not meet several deadlines, then met with Tom to discuss SP, rather than provide updated cost details on his priority tasks, _as requested at Monday’s **Workgroup Leads** meeting. 
- The relative priority of this meeting with Tom, as compared to other workgroups depending on his delayed other issues, should have been discussed at Monday’s Workgroup leads meeting & resolved, esp. with things that are running late.
- Example 2: You attempted to apply a due date for issues that have been unresolved for several weeks within the **quant_workgroup** because they are running late, have not been updated, and are an increasingly important priority.
- No accurate cost (time) estimate has been provided by the Workgroup.
- Questions about one of the issues had not been prioritized by the Workgroup for discussion at any standing meetings (& the P.I. has not been mentioned on the issue to alert her to the questions). 
- The Workgroup Lead simply adjusted the due date out a full month without providing adequate justification for it requiring a full month of work. Or, _more importantly_ clarifying the questions that would enable some work to proceed now.

_PURPOSE of SOP Recommendation 2:_
a. When combined with Recommendation 1, this should lead to better consideration of cost (time) estimates, dependencies, efficiency in identifying potentially parallel workstreams, when determining priorities and assigning due dates.
b. This should lead to greater overall stability & clarify between HQ and Workgroups, and within and across Workgroups about what to work on when.

@staceypark 
1. Please add this to tomorrow’s **Support Workgroup Leads** agenda and walk through the recommendations, purpose, and examples to orient/train folks.
2.  Please filter by _assignee_ to determine the accurate cost (time) estimates for each outstanding **quant_workgroup_** task, so that they can all be discussed/clarified at tomorrow’s **quant_workgrouo meeting** 
- With better, clear, updated information about these tasks:
a. We can determine what Savet and Andrew each do in parallel, and when handoffs will occur. 
b. We should then be ready to update DUE DATES for **quant_workgroup** at Monday’s **Workgroup Leads** meeting, considering how to divvy-up time on various tasks throughout the short week next week, and the subsequent week.

Thanks!

**Comment by ritahitching** [comment]
@lzim completed  OSF: https://osf.io/adfx3/

**Comment by lzim** [comment]
> @lzim completed OSF: https://osf.io/adfx3/

@ritahitching thanks, but can you be more specific? I’m not sure which OSF task you mean in the hq training project at the link 😆

**Comment by ritahitching** [comment]
@lzim :see_no_evil: 
link to OSF: https://osf.io/adfx3/ 
 https://osf.io/adfx3/wiki/Typical%20Weekly%20Meeting%20Calendar%20Attendees/

**Comment by ritahitching** [comment]
@lzim @jessfroe @staceypark 
I started the **TeamPSD Training Manual Draft V 0.1** in **OSF** using the word document @jessfroe started. If there are any existing additional resources that you'd like me to add please email them to me or direct me as to where I can find them on GitHub or TeamPSD folder - don't want to reinvent the wheel. :pray:
https://osf.io/adfx3/wiki/TeamPSD%20Training%20Manual%20Draft%20V%200.1/

**Comment by staceypark** [comment]
@lzim sounds good I can show @ritahitching where to find those

**Comment by staceypark** [comment]
@lzim I created a CONSORT checklist for each grant in its respective OSF project.
I had some questions & notes about each which you can find at the top of each checklist.

**r01 consort checklist**: https://osf.io/8pzxu/wiki/consort_checklist/
**iir consort checklist**: https://osf.io/p7g8f/wiki/consort_checklist/

**Comment by ritahitching** [comment]
@staceypark @lzim 
Currently populating mtl.how/team_tracker resource cards with links S01-S06
Note S02 has 2 versions of the cheatsheet: I used the most recent version 

1. mtl_how_data_cheatsheet.pdf	updated **4 months ago**
2. mtl_how_live_cheatsheet.pd updated **14 days ago**

**Comment by lzim** [comment]
Happy Friday HQ! 🏖

#### _Top 3 Priorities for things LATE or due COB_

**_1. quant_workgroup_**
- [ ] EBP Course data 
- Cross-ref **Original Issue ** #462 **9/16 EBPsy and EBPharm course data** - LZ will try to manage this thread and provide guidance today.
- Cross-ref #615 **EBP Reach Dose calculation** THIS IS DUE ASAP, some components may be completed now, unfortunately still has the October Due date that was assigned by Savet (now a violation of the SOP updated DUE DATE assignment during meetings policy)
- Cross-re #571 **EBP Reach Initiate & Dose definitions documentation** where much of the details about how to operationalize have been provided **on OSF** here: https://osf.io/z7y8m/wiki/home/
- NOTE: For documentation #571 dependencies - SOME NEEDED for CONSORT, CT.Gov/ART registration and  #559 **ro1_protocol** and #561 **iir_protocol** 
@saveth also cross-ref’d #630, she also cross-ref’d #424 **p-charts**, which she then closed, and created #672, #674, and #675, and she also cross-ref’d #476 9/24 - **EBP Initiate measures** used for _site selection_

**_2. documentation scheduled promised to OMHSP ICs_**

- **I’m hoping you’ve been prompting folks all week 😬...But, I haven’t gotten any prompts, could it be because this workflow was not captured in the #hqhuddle issue? 🌸

- NOTE: These are all the currently open cards on the **document_tracker** that are due to be closed by COB today (from last week and this week)
1. Each documentation point person needs to review the **meta_card** and the *session_card checklist,** check off that each has been reviewed, and provide a link to the current documentation for HQ review.
2. HQ reviews and closes the card.

_Jane & Debbie: **describe_learners**
- [ ] s05 - #496 
- [ ] s06 - $507 
- [ ] s07 - #506 

_Tom & Lindsey: **detail_facilitators**
- [ ] s05 - #497  - SAY: No input from Tom, yet - Need s05_checklist - LZ will produce
- [ ] s06 - $518  - SAY: Tom added Systems Stories - LZ will add _fidelity_ intro - have a s06_checklist (LZ will rename the file)
- [ ] s07 - #517  - SAY: Tom was supposed to add BC details, but hasn’t yet - LZ will add _fidelity_intro - have a s07_checklist

 _Stacey & Rita at HQ_ **document_team**
- [ ] s05 - #534  
- [ ] s06 - #529 
- [ ] s07 - #528  

_Savet, Andrew & Lindsey_ **disseminate_scientists_va**
- [ ] s05 - #572 
- [ ] s06 - $571 
- [ ] s07 - #570  

_Jane & Debbie_ **depend_products**
- [ ] s05 - #558 
- [ ] s06 - $553 
- [ ] s07 - #552 

_**3. Manuscripts fully organized/drafted, with references, knitted in APA style & circulated to co-authors by COB with goal of submitting by COB Monday**_
- [ ] Swap, @lzim  & @staceypark today **9/16 COB - r21_mentee_first_person_ajcp - https://osf.io/6brn3/ #563
- [ ] David, @lzim  & @staceypark today **9/16 COB - r21_facilitate1_gmb - https://osf.io/mjdx7/ #564**

#### Other Misc. 
@staceypark OR @ritahitching 
_from above - needs to be added to SOP_
- [ ] # HQ Decisions Re: #hqhuddle and priorities 
- [ ]  1. trackers require current/accurate cost estimates in order to evaluate TeamPSD dependencies and priorities
- [ ] 2. Due Dates on issues should not be edited on the kanban/tracker outside of a Workgroup Leads, Support Workgroup Leads, or Workgroup meeting (with only rare exceptions discussed/documented on the GH issue)

@staceypark 

_points of clarification_
- [ ]  **On hold until we get through 1) this week’s documentation (Tom), and this week’s top data priorities (Andrew) - Need before launch, will not incorporate in training: Coordinate with @TomRust and @holbrooa** to test **Team 4** on **9/5 prototype Data UI: Use PC diagnoses from Jodie for SP instead of using stop codes #596
- [ ]  R01 CT.Gov and ART registration Cross-ref #606 and #607 -  We need the same details populated in our CT registration and ART registration - We want to make sure everything we already have is completed on each (CT & ART), then work to fill in the remainder either on Monday for #manuscriptmonday focused on _r01_protocol_ and _iir_protocol for Tuesday.

@ritahitching 
_from above - needs to be captured **outside** this issue on TeamPSD_
- [ ] Monitors table needs to be in Team PSD with the Workstations file
- [ ] Resources wishlist needs to be tracked probably in Team PSD. > pavir_expenses in a new folder called “upcoming_expenses”

_please print_
- [ ] The HQ attendee meeting schedule. I’d like to have us all pin it up on the board near our office, so we can keep track of it.

_monitors_
- [ ] Yes, please do email the list to Mary & Doug and cc Stacey and I
- [ ] Would you be able to try to install my two new monitors (box 1 and 2, in my office) on my sit/stand desk today? It makes the most sense to do that while I’m out. It would require working with VESA mount, but we want to save the stands/boxes for the move, as it will be a different set-up in the morning.

@lzim 
- As you can see above, while at #SIRC2019 I’ll be focusing on my 
1. **quant_workgroup EBP course** thread/clarification
2. **detail_facilitate** documentation for s05, s06, and s07 due by COB, and **_r21_mentee_** and 3. **_r21_facilitate_gmb_** manuscripts
4. I will also be working with James a bit on some of the DEMO stuff for our two presentations tomorrow.

Thanks All!

**Comment by staceypark** [comment]
FYI @lzim GitHub is not working in VA. I double checked on all the browsers just in case

**Comment by lzim** [comment]
FYI: @staceypark 

> GitHub not working in VA
- I was concerned this may be the immediate result of the NSD ticket
- However, @ritahitching is using GitHub, please explain?

@ritahitching - glad you mentioned this - please STOP (quote references below) 😄 
- We do NOT necessarily need to fully populate the **team_tracker** _prototype_ for those sessions
- @jessfroe created **resource_cards**  but we said we would NOT go back and fully populate sessions that we have already trained on (s01-s05) because we plan to make the **team_tracker** a **private_tracker**

- THIS

> @staceypark @lzim
> Currently populating mtl.how/team_tracker resource cards with links S01-S06
> Note S02 has 2 versions of the cheatsheet: I used the most recent version
> 
> 1. mtl_how_data_cheatsheet.pdf	updated **4 months ago**
> 2. mtl_how_live_cheatsheet.pd updated **14 days ago**

- AND THIS:
> @staceypark @lzim
> Completed populating mtl.how/team_tracker resource cards - noticed some discrepencies - > > > please confirm if these are ok or if I should update, and if so how
>
> Inconsistent links to other platforms
> Pre and Post checklists not matching session number
> Say guides missing


> FYI @lzim GitHub is not working in VA. I double checked on all the browsers just in case



> FYI @lzim GitHub is not working in VA. I double checked on all the browsers just in case

**Comment by lzim** [comment]
@staceypark 

- Re: GitHub down:  You’ll need to open another NSD help desk ticket with the additional issue and cross-ref your original ticket.

**Comment by lzim** [comment]
> FYI @lzim GitHub is not working in VA. I double checked on all the browsers just in case.
> 
> @ritahitching and I are planning to catch WiFi around the welcome center.
> 
> I can come back in periodically to check up on my email though

@staceypark and @ritahitching 
- Before working on WiFi...you need to work as hard as you can to resolve this issue ASPA on behalf of the team.
- **IF NO ONE FROM TEAM PSD can use GitHub from VA today** then priorities have shifted:
1. make sure to get a new NSD ticket going on the NEW issue ASAP
2. follow-up with the GitHub folks you contacted first earlier in the week _BEFORE_ opening an NSD HelpDesk ticket.
3. Please make sure all NSD tickets and GitHub communications are backed up in TeamPSD for others to track.

**Comment by staceypark** [comment]
@lzim we're both using wifi/data but ok I'll reach out to GitHub and NSD as my first thing

**Comment by staceypark** [comment]
I updated the GH ticket with the newest issue (still have not heard back), created a new ticket with NSD, and updated James ticket on #640 with the issue as well.

They did tell me James's ticket got assigned to someone yesterday but could not tell me who.

**Comment by ritahitching** [comment]
@lzim your 2 monitors are installed - need to test them. Are you remote logging in?

**Comment by ritahitching** [comment]
@LZ @staceypark  @jessfroe  the Resources Wish List is now located in the:
TeamPSD/Pavir Expenses/Upcoming Expenses/ teamPSDwishlist.xls

**Comment by staceypark** [comment]
@ritahitching thanks for working on this! can you make sure to name the folder and document using all lowercase and underscores following teampsd naming convention to make all of our files machine & human readable.

**Comment by staceypark** [comment]
@lzim document tracker:

- depend: I pingged on the 3 issues. It looks like we only have S06 (almost done) so far.
- SAY: @TomRust won't be back until Tuesday, so we won't have his pieces before that.
- SEE: s05 & s07 are done from Jane/Debbie's end. I'm updating the gifs right now so that I can completely close the issue. s06 we said on the issue that @jessfroe would take a stab at aligning the SEE/SAY first before handing it over. Not sure if @jessfroe got around to it though. Regardless, either I can work on it or we can move it down a week.

**Comment by ritahitching** [comment]
@lz @jessfroe @staceypark 
I contributed some to the Team PSD Training Guide but it needs more materials added to it in OSF hqtraining https://osf.io/adfx3/

**Comment by ritahitching** [comment]
@lzim @staceypark @jessfroe 
My carryover task into week commencing Sep 16 is:
1. finish populating fidelity survey
- all others done. 

I'm holding a  :wine_glass: - have a great weekend everyone, and thanks for your help during my first full week.

**Comment by lzim** [comment]
Monday | Tuesday | Wednesday | Thursday | Friday
-- | -- | -- | -- | --
8AM Workgroup Leads |   |   |   |  
9AM-12noon #manuscriptmondays |   |   |   |

**Comment by ritahitching** [comment]
@lzim - I hope this is clearer - let me know if you want any additional edits
@staceypark 



Monday | HQ Lead | HQ Attendees |   | Tuesday | HQ Lead | HQ Attendees |   | Wednesday | HQ Lead | HQ Attendees |   | Thursday | HQ Lead | HQ Attendees |   | Friday | HQ Lead | HQ Attendees
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
8:00am  to 9:00am Workgroup Leads |   | LZ RH SP |   | 8:00am  to  9:00am qual coding workgroup |   | LZ SP |   | 9:30am to 11:30am intersession prep |   | LZ SP RH |   | 8:00am to 8:55am support workgroups |   | LZ SP |   | 9:30 am to 11:30 am MTL Facilitate |   | LZ RH SP
9:00am  to  12:00pm Manuscript Monday |   | LZ RH SP |   | 10:00 to  10:55am manuscript group |   | LZ SP RH |   | 11:00am to 12:00pm all staff meeting |   | LZ RH SP SM SH |   | 10:00am to 10:55 am quant work groups |   | LZ SP RH |   | 12:00pm to 12:30pm HQ Huddle |   | LZ RH SP
12:30pm to 1:00pm HQ Huddle |   | LZ RH SP |   | 11:00am to 1:00pm psychotherapy tool |   | LZ SP RH |   | 12:00pm to 12:30pm HQ Huddle |   | LZ RH SP |   | 1:00am to 2:00pm HQ workgroup |   | LZ SP RH |   | 4:00 pm to 4:30pm HQ Huddle |   | LZ RH SP
4:00 pm to 4:30pm HQ Huddle |   | LZ RH SP |   | 1:00pm to 2:00pm TeamPSD Lunch |   | LZ RH SP SM SH |   | 4:00 pm to 4:30pm HQ Huddle |   | LZ RH SP |   | 12:00pm to 12:30pm HQ Huddle |   | LZ RH SP |   |   |   |  
  |   |   |   | 4:00 pm to 4:30pm HQ Huddle |   | LZ RH SP |   |   |   |   |   | 4:00 pm to 4:30pm HQ Huddle |   | LZ RH SP |   |   |   |  


- [ ] Can we discuss the afternoon and wednesday #hqhuddle timing at our next meeting please?
